static/js/event/event.ts

```diff
@@ -18,14 +18,49 @@
  * Disaster events
  */
 
+import axios from "axios";
 import React from "react";
 import ReactDOM from "react-dom";
 
-import { Page } from "./page";
+import { EventPage, Page } from "./page";
+import { Property } from "./types";
+
+function parseApiResponse(resp): Array<Property> {
+  // Parses response from triples API for later rendering
+  const parsedResponse = [];
+  for (const [property, connectedNodes] of Object.entries(resp)) {
+    for (const nodes of Object.values(connectedNodes)) {
+      parsedResponse.push({ name: property, values: nodes });
+    }
+  }
+  return parsedResponse;
+}
 
 window.onload = () => {
-  ReactDOM.render(
-    React.createElement(Page),
-    document.getElementById("main-pane")
-  );
+  const dcid = document.getElementById("node").dataset.dcid;
+  const nodeName = document.getElementById("node").dataset.nn;
+  const propertiesPromise = axios
+    .get(`/api/node/triples/out/${dcid}`)
+    .then((resp) => resp.data)
+    .catch(() => {
+      return 0;
```

Comment:
since the error case is already being handled in line 60, I don't think we need this catch block.

Sidenote, we probably wouldn't have wanted to return 0. In the`.catch`, we would want to return something that is of the same type as what is returned in the `.then`

Reply:
Thanks for the tip! The triples API call is now being done in flask, so this section has been removed.